draft-ietf-cats-framework.md

```diff
@@ -532,17 +540,17 @@ If the CATS framework is implemented using an hybrid model, the metric can be di
 
 A C-PS computes paths that lead to Egress CATS-Forwarders according to both service and network metrics that were advertised. A C-PS may be collocated with an Ingress CATS-Forwarder (as shown in {{fig-cats-example-overlay}}) or logically centralized (in a centralized model or hybrid model).
 
-This document does not specify any algorithm for path computation and selection purposes to be supported by C-PSes. However, it is expected that a service request or local policy may feed the C-PS computation logic with Objective Functions that provide some information about the path characteristics (e.g., in terms of maximum latency) and the selected service contact instance.
+This document does not specify any algorithm for path computation and selection purposes to be supported by C-PSes. However, it is expected that a service request or local policy may feed the C-PS computation logic with Objective Functions that provide some information about the path characteristics (e.g., in terms of maximum latency) and the preference in selecting the service contact instance.
```

Comment:
```suggestion
This document does not specify any path computation and instance selection algorithm running in C-PSes. However, it is expected that a service request or a policy may feed the C-PS computation logic with Objective Functions that provide the optimization goal or preference in selecting the service contact instance.
```

Reply:
Any reason to remove the example ("e.g., in terms of maximum latency")? Otherwise, we have text that is too generic? I would keep the example here. Also, may delete "preference" as this can be part of the goal. 

I still prefer the OLD text.

Reply:
> Any reason to remove the example ("e.g., in terms of maximum latency")? Otherwise, we have text that is too generic? I would keep the example here. Also, may delete "preference" as this can be part of the goal.
> 
> I still prefer the OLD text.

Objective Functions does not provide information of the path charateristics. The path characteristics is the input of the Objective Function.

Reply:
I was not questioning that part of the modif :-)

Reply:
To me, we may need to use some straightforward wording for better understanding. Objective function introduces unnecessary complexities to me.